netdisco/discoverables/yamaha.py

```diff
@@ -5,7 +5,7 @@
 class Discoverable(SSDPDiscoverable):
     """Add support for discovering Yamaha Receivers."""
 
-    INCOMPATIBLE_MODELS = set('N301')
+    COMPATIBLE_MODELS = "RX-V"
```

Comment:
From linked bug:

> The way this implementation discovers yamaha platform is not taking into consideration newer platforms of Yamaha MusicCast.

With this PR, it will not take into consideration old platforms? 🤔

Reply:
Oh I see now the check is also inverted, check.